src/main/resources/application.properties

```diff
@@ -1,2 +1,3 @@
 server.port=5000
-spring.data.mongodb.uri=mongodb+srv://<username>:<password>@cluster0-nomfh.mongodb.net/InternHome
+spring.data.mongodb.uri=mongodb+srv://db_user:<password>@cluster0-nomfh.mongodb.net/InternHome
+spring.data.mongodb.database = InternHome
```

Comment:
Why is this needed?

Reply:
because the URI is necessary for the connection to the database, but this section "/ InternHome" is not necessary.